python/tvm/relay/qnn/op/qnn.py

```diff
@@ -72,3 +72,62 @@ def requantize(data,
                             output_zero_point,
                             rounding,
                             out_dtype)
+
+
+def quantize(input_data,
```

Comment:
what about simply name the input tensor as `data`, because it sounds strange to div it with output_* parameters.